### ISSUE TITLE
Can we detect APIs of data brokers?

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/privacy-pioneer#16]
So far we focused a lot on intercepting HTTP requests between a site and a user. However, there is also the connection between the site and third parties (without involving the user at all). In particular, a site could share data on the backend with data brokers. Can we detect whether a site is integrated with a data broker API?

The first point here is that there are various lists of data brokers as they are now required to register there activity per the Vermont data broker law and similar other laws. 
- [List per the Vermont data broker law](https://bizfilings.vermont.gov/online/DatabrokerInquire/DataBrokerSearch)
- [California Data Broker Registry (probably even better)](https://oag.ca.gov/data-brokers?combine=&order=field_dbr_name&sort=asc)
- [Big-Ass-Data-Broker-Opt-Out-List](https://github.com/yaelwrites/Big-Ass-Data-Broker-Opt-Out-List)

Now, I could imagine that we are not able to find HTTP requests between users and data brokers directly, but between the sites and data brokers. But I do not exactly know how selling data to data broker works. Here is a [general overview](https://clearcode.cc/blog/what-is-data-broker/) that is quite good. I also will try to ask some people in adtech I know. 

We could try to focus on some big data brokers instead of comprehensive coverage. For example, here is an [API reference from acxiom how to send Person information](https://developer.myacxiom.com/code/api/documents/person). I also found some explanations on the [lotame site for monetizing data](https://www.lotame.com/how-to-monetize-your-data/). They are sort of a data marketplace. [Lotame PDX for Sellers](https://www.lotame.com/pdx/sellers/) looks interesting as well. I am not sure if we could sign up for services like these and try to understand better how they work. Or, obviously easier, find publicly available documentation. 

The bottom line is that it would be great to identify a site's integration with data brokers. And my sense is that we would need to do some more detective work to understand better how these data brokers work.

1. Are they involved in HTTP requests between users and the site?
2. Can we detect their API connection to a site at the backend (which requires us to understand how sites integrate with those data brokers).
